sklearn/utils/_mask.py

```diff
@@ -1,11 +1,18 @@
 import numpy as np
 from scipy import sparse as sp
+from contextlib import suppress
 
 from . import is_scalar_nan
 from .fixes import _object_dtype_isnan
 
 
 def _get_dense_mask(X, value_to_mask):
+    with suppress(ImportError, AttributeError):
```

Comment:
I'm guessing the `AttributeError` is for older versions of pandas that do not have `pd.NA`? If so, may we leave a comment about it?

Reply:
Exactly! Suppressing `AttributeError` is for older versions of pandas that do not have `NA`. Added a comment as suggested.